commit 9e8b044d982e132cf35a106a1cc0cf7e77b27f7c
Author: Mauricio Collares <mauricio@collares.org>
Date:   Thu Apr 15 20:33:21 2021 -0300

    Disable manual sections using optional packages not yet in Nixpkgs

        * normaliz.lib depends on normaliz.
        * polymake.lib depends on topcom.
        * recover.lib depends on bertini.
        * sing4ti2.lib depends on 4ti2.
        * tateProdCplxNegGrad.lib uses multigrading.lib, which depends on 4ti2.

diff --git a/doc/singular.doc b/doc/singular.doc
index 64b969d39..e704f95f0 100644
--- a/doc/singular.doc
+++ b/doc/singular.doc
@@ -407,7 +407,6 @@ LIB "all.lib";
 * nfmodsyz_lib:: Syzygy modules of submodules of free modules over algebraic number fields
 * noether_lib:: Noether normalization of an ideal
 * normal_lib:: procedure for normalization
-* normaliz_lib:: integral closure, normalization for monomial ideals, toric ideals
 * pointid_lib:: factorized lex GB of the vanishing ideal of a set of points
 * primdec_lib:: procedures for primary decomposition
 * primdecint_lib:: primary decomposition over the integers
@@ -416,7 +415,6 @@ LIB "all.lib";
 * reesclos_lib:: Rees Algebra and integral closure of an ideal
 * rstandard_lib:: Janet bases and border bases for ideals
 * sagbi_lib:: Subalgebras bases Analogous to Groebner bases for ideals
-* sing4ti2_lib:: interface to program 4ti2
 * symodstd_lib:: Groebner bases for symmetric ideals
 * toric_lib:: toric ideals
 @end menu
@@ -521,10 +519,6 @@ LIB "all.lib";
 @node normal_lib
 @subsection normal_lib
 @c lib normal.lib
-@c ---------------------------------------------------------
-@node normaliz_lib
-@subsection normaliz_lib
-@c lib normaliz.lib tag:normaliz
 @c ----------------------------------------------------------
 @node pointid_lib
 @subsection pointid_lib
@@ -558,10 +552,6 @@ LIB "all.lib";
 @subsection sagbi_lib
 @c lib sagbi.lib
 @c ---------------------------------------------------------
-@node sing4ti2_lib
-@subsection sing4ti2_lib
-@c lib sing4ti2.lib tag:sing4ti2
-@c ----------------------------------------------------------
 @node symodstd_lib
 @subsection symodstd_lib
 @c lib symodstd.lib
@@ -873,7 +863,6 @@ iniD, reslist, sumlist, dividelist, createlist
 * solve_lib:: procedures to solve polynomial systems
 * triang_lib:: procedures for decomposing zero-dimensional ideals
 * ntsolve_lib:: one real solution of polynomial systems (Newton iteration)
-* recover_lib:: Hybrid numerical/symbolical algorithms
 * rootisolation_lib:: real root isolation with intervals
 * signcond_lib:: computing realizable sign conditions
 * zeroset_lib:: procedures for roots and factorization
@@ -904,10 +893,6 @@ iniD, reslist, sumlist, dividelist, createlist
 @subsection ntsolve_lib
 @c lib ntsolve.lib
 @c ---------------------------------------------------------
-@node recover_lib
-@subsection recover_lib
-@c lib recover.lib tag:bertini
-@c ----------------------------------------------------------
 @node rootisolation_lib
 @subsection rootisolation_lib
 @c lib rootisolation.lib
@@ -1108,7 +1093,6 @@ but not for serious computations.
 * cimonom_lib:: complete intersection for toric ideals
 * gfan_lib:: A gfanlib interface for Singular
 * gitfan_lib:: Compute GIT-fans
-* polymake_lib:: interface to TOPCOM
 * realizationMatroids_lib:: Realizability for Tropical Fan Curves
 * tropical_lib:: interface to gfan
 * tropicalNewton_lib:: Newton polygons in tropical geometry
@@ -1125,10 +1109,7 @@ but not for serious computations.
 @node gitfan_lib
 @subsection gitfan_lib
 @c lib gitfan.lib
-@c ----------------------------------------------------------
-@node polymake_lib
-@subsection polymake_lib
-@c lib polymake.lib tag:topcom
+
 @c ----------------------------------------------------------
 @node realizationMatroids_lib
 @subsection realizationMatroids_lib
@@ -1219,7 +1200,6 @@ Comments should be send to the author of the library directly.
 * stanleyreisner_lib:: T1 and T2 for a general Stanley-Reiser ring
 * swalk_lib:: Sagbi Walk Conversion Algorithm
 * systhreads_lib:: multi-threaded objects
-* tateProdCplxNegGrad_lib:: sheaf cohomology on product of projective spaces
 * VecField_lib:: vector fields
 @end menu
 @c ----------------------------------------------------------
@@ -1310,10 +1290,6 @@ Todos/Issues:
 @subsection systhreads_lib
 @c lib systhreads.lib
 @c ---------------------------------------------------------
-@node tateProdCplxNegGrad_lib
-@subsection tateProdCplxNegGrad_lib
-@c lib tateProdCplxNegGrad.lib
-@c ---------------------------------------------------------
 @node VecField_lib
 @subsection VecField_lib
 @c lib VecField.lib
